### PR TITLE
Basic template for Betomic

### PR DESCRIPTION
**Pull request by yonatan-rojas**
## Solution

- created scaffolding of betomic library

## Test

`Not yet`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "git+https://github.com/ExponentialEducation/betomic.git"
+  },
+  "author": "ExponentialEducation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ExponentialEducation/betomic/issues"
+  },
+  "homepage": "https://github.com/ExponentialEducation/betomic#readme",
+  "dependencies": {},
```
Comment (coderdiaz) [inline package.json]: Remove this line

File: packages/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+
+const Button: React.SFC<{ text: string }>
```
Comment (coderdiaz) [inline packages/Button/index.tsx]: Change `{ text: string }` to Props Interface.

File: packages/HelloWorld/index.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+
+const HelloWorld: React.SFC<{ text: string }>
```
Comment (coderdiaz) [inline packages/HelloWorld/index.tsx]: Change `{ text: string }` to Props Interface.

**Comment by pitakill** [comment]
As a side comment, Do you have considerate Svelte for the new developments in the frontend area?

Looks pretty amazing the footprint of the build and the developer experience is as much alike with frameworks like React and Vue

**Comment by coderdiaz** [comment]
> As a side comment, have you considered Svelte for the new developments in the frontend area?
> 
> Looks pretty amazing the footprint of the build and the developer experience is as much alike with frameworks like React and Vue

@pitakill Yes, but we have a minimal time for deploy this project and use Gatsby + own React UI toolkit it's a good approach at the moment. We are considering use Svelte in another non-priority project.

**Pull request closed without merging**